### PR TITLE
add Kagla.finance

### PR DESCRIPTION
**Pull request by artat9**
##### Twitter Link:
https://twitter.com/kagla_fi

##### List of audit links if any:
N/A(in progress)

##### Website Link:
https://kagla.finance/

##### Logo (High resolution, preferably in .svg and .png, for application on both white and black backgrounds. Will be shown with rounded borders):
here: https://github.com/kagla-finance/media-kit/tree/main/BI%20for%20Web

##### Current TVL:
```
------ TVL ------
astar                     1.89 M

total                    1.89 M
```

##### Chain:
astar

##### Coingecko ID (so your TVL can appear on Coingecko): (https://api.coingecko.com/api/v3/coins/list)
not yet

##### Coinmarketcap ID (so your TVL can appear on Coinmarketcap): (https://api.coinmarketcap.com/data-api/v3/map/all?listing_status=active,inactive,untracked&start=1&limit=10000)
not yet

##### Short Description (to be shown on DefiLlama):
The biggest native stable swap on Astar Network boosting.

##### Token address and ticker if any:
not yet

##### Category (full list at https://defillama.com/categories) *Please choose only one:
Dexes

##### Oracle used (Chainlink/Band/API3/TWAP or any other that you are using):
no

##### forkedFrom (Does your project originate from another project):
Curve

##### methodology (what is being counted as tvl, how is tvl being calculated):
Total amount of stable coins locked in the Liquidity pools. All stable coin's price(price of USDC, USDT, DAI and BUSD) are calculated as $1.
https://github.com/kagla-finance/kagla-api/blob/main/src/contracts/poolInfo/calculator.ts#L7

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/kagla/index.js exports TVL: 
        
 
 
    astar                     1.99 M
    
    total                    1.99 M

File: projects/kagla/index.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+function fetch() {
+  return astar();
+}
+
+module.exports = {
+  timetravel: false,
+  misrepresentedTokens: true,
+  astar: {
+    fetch: astar,
```
Comment (g1nt0ki) [inline projects/kagla/index.js:16]: Can this be calculated using on-chain calls instead? We've got a guide on how to use our SDK here: https://docs.llama.fi/list-your-project/submit-a-project

Comment (artat9) [reply projects/kagla/index.js:16]: Sure. I'll try it later!

**Comment by llamatester** [comment]
The adapter at projects/kagla/index.js exports TVL: 
        
 
 
    astar                     10.49 M
    
    total                    10.49 M

**Pull request merged**